### PR TITLE
[PAN-3005] New API parameter format.

### PR DESCRIPTION
**Pull request by bgravenorst**
## PR description

The parameters for `priv_createPrivacyGroup` are listed in an object. Additionally, the `name` and `description` parameters are optional.

## Fixed Issue(s)
PAN-3005

### CODE REVIEW

**Comment by bgravenorst** [comment]
@MadelineMurray, I documented the Parameters section similarly to how we do it in `debug_traceBlockByHash` because it now uses an object.  So I'm not 100% sure the format is correct.

File: docs/Reference/Pantheon-API-Methods.md [hunk 1]
```diff
@@ -4070,23 +4070,25 @@ Creates a privacy group containing the specified members. Members are specified
 
 **Parameters**  
 
-`array of data` - Array of members specified by Orion public keys 
+`Object` - Request options:
 
-`string` - Privacy group name 
+* `addresses`: `array of data` - Array of members specified by Orion public keys
```
Comment (NicolasMassart) [inline docs/Reference/Pantheon-API-Methods.md]: blank lines not needed here

File: docs/Reference/Pantheon-API-Methods.md [hunk 2]
```diff
@@ -4070,23 +4070,25 @@ Creates a privacy group containing the specified members. Members are specified
...
 
-`array of data` - Array of members specified by Orion public keys 
+`Object` - Request options:
 
-`string` - Privacy group name 
+* `addresses`: `array of data` - Array of members specified by Orion public keys 
 
-`string` - Privacy group description
+* `name`: `string` - Privacy group name. Optional.
+
```
Comment (NicolasMassart) [inline docs/Reference/Pantheon-API-Methods.md]: blank lines not needed here

**Review by NicolasMassart — APPROVED** [review]
LGTM, just two useless blank lines that you could remove

**Review by iikirilov — APPROVED** [review]
LGTM

**Pull request merged**